samples/nrf9160/pdn_client/README.rst

```diff
@@ -0,0 +1,124 @@
+.. _pdn_client_sample:
+
+nRF9160: PDN Client
+##################
+
+The PDN client sample demonstrates how to open and connect to PDN with the nRF9160 modem.
+
+Overview
+********
+
+The AT Client sample create, connect and close to PDN connection.
+
+
+Requirements
+************
+
+* The following development board:
+
+  * |nRF9160DK|
+
+* .. include:: /includes/spm.txt
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/pdn_client`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Connecting to the nRF9160 DK board with LTE Link Monitor
+========================================================
+
+To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
+
+#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
```

Comment:
This documentation is a copy-paste from AT Client, how does it relate to this sample?

Reply:
it is copied from at_client sample and need to be updated... I will fix this.